frontend/app/services/IdentityService.scala

```diff
@@ -162,7 +162,7 @@ class IdentityApi(val wsClient: WSClient, implicit val ec: ExecutionContext) {
   def getUserPasswordExists(headers:List[(String, String)], parameters: List[(String, String)]) : Future[Boolean] = {
     val endpoint = "user/password-exists"
     val url = s"${Config.idApiUrl}/$endpoint"
-    wsClient.url(url).withHttpHeaders(headers: _*).withQueryStringParameters(parameters: _*).withRequestTimeout(1000 milli).get().map { response =>
+    wsClient.url(url).withHttpHeaders(headers: _*).withQueryStringParameters(parameters: _*).withRequestTimeout(1.second).get().map { response =>
```

Comment:
the postfix notation needs a language import adding, so I changed to the slightly more conventional way.  Also I decided 1 second is clearer than 1000 ms